hadoop-ozone/fault-injection-test/mini-chaos-tests/src/test/java/org/apache/hadoop/ozone/TestMiniChaosOzoneCluster.java

```diff
@@ -91,6 +98,16 @@
           "--failureInterval will be removed in later versions.")
   private static int failureInterval = 300; // 5 minute period between failures.
 
+  @Option(names = {"-r", "--replication-type"},
```

Comment:
Across various commands we have been adding two options to facilitate EC:

* `-t --type` - for RATIS or EC
* `-r --replication` - the replication that goes with the type - eg ONE, THREE, rs-3-2-1024k etc.

It would be best to try to keep that convention here and add both options. Then remove the hardcoded `rs-3-2` added later in the class, as it would allow it to be more flexible in the future.

Reply:
`-t` is already taken for number of threads.

However, you can add the `FreonReplicationOptions` (which omits `-t` for the same purpose):

https://github.com/apache/ozone/blob/c2512588322677afcb0118c9ccb0057f01b877b3/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmKeyGenerator.java#L64-L65

and then use it:

https://github.com/apache/ozone/blob/c2512588322677afcb0118c9ccb0057f01b877b3/hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java#L128-L129

(adapt to this class by replacing `getConf()` with `configuration`).